ci/lint/06_script.sh

```diff
@@ -6,9 +6,9 @@
 
 export LC_ALL=C
 
-if [ "$TRAVIS_EVENT_TYPE" = "pull_request" ]; then
-  test/lint/commit-script-check.sh $TRAVIS_COMMIT_RANGE
-fi
+#if [ "$TRAVIS_EVENT_TYPE" = "pull_request" ]; then
+#  test/lint/commit-script-check.sh $TRAVIS_COMMIT_RANGE
+#fi
```

Comment:
We don't use the scripted-diff system yet, but I wonder if we should leave this condition for Travis and determine whether Actions provides a similar variable. Not high-priority, but it may avoid some hair-pulling in the future.

Maybe adding a comment will be enough for now.

Reply:
Fixed.